docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -42,6 +34,20 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 100 GB SSD disk or more
 - 64-bit CPU architecture
```

Comment:
minor: x86-64 CPU architecture would be more appropriate since we don't support ARM/aarch64

Reply:
Changed